book/website/reproducible-research/reviewing/reviewing-checklist.md

```diff
@@ -1,40 +1,180 @@
+(rr-checklist-for-code-review)=
 ## Checklist for code review process
 
-The following presents some possible checklists for both the coder and the reviewer, as part of a formal review process.
+This section presents some checklists for both the coder and the reviewer, as part of a formal review process.
+The reviewer checklists are split into two categories: one for the whole program, and one for individual files or proposed changes.
+
+The lists are created with a focus on good software engineering practice and are intended to be a source of inspiration.
+When assessing the checklists, it is recommended to consider to what extent the item mentioned is implemented.
+Some items on the lists may not apply to your project or programming language, in which case they should be disregarded.
+
+In all cases, the goal is to use your programming experience to figure out how to make the code better.
 
 ## Table of contents
 
 - [For the coder](#For_the_coder)
 - [For the reviewer](#For_the_reviewer)
+- [Program level checklist](#Program_level_checklist)
+- [File/Change level checklist](#File/Change_level_checklist)
 
 <a name="For_the_coder"></a>
```

Comment:
Naive question. Is this `<a name=` HTML tag recognized for cross-linking by the book assembly software?

Reply:
Good question. The[ handbook style guide](https://the-turing-way.netlify.app/community-handbook/style/style-crossref.html) does not mention something about the HTML headings. I am wondering why the inline HTML has been used in several sections of the book. My markdown linter recommends removing inline HTML. So I removed those from files in the code review section.